modules/pam_unix/support.c

```diff
@@ -515,6 +518,89 @@ int _unix_comesfromsource(pam_handle_t *pamh,
 #include <sys/types.h>
 #include <sys/wait.h>
 
+#ifdef USE_PWACCESS
+int _unix_verify_pwaccess(pam_handle_t *pamh, const char *user,
+			  int *daysleft)
+{
+    int  r, retval = PAM_SYSTEM_ERR;
+    bool pwchangeable = false;
+    long dl = -1;
+
+    D(("called."));
+
+    r = pwaccess_check_expired(user, &dl, &pwchangeable, NULL);
```

Comment:
Looks like `pwaccess_check_expired` can return a textual error. Is there any reason why this feature is not used here?